cmd/bridgec/internal/client.go

```diff
@@ -0,0 +1,119 @@
+package internal
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/threefoldtech/rivine/pkg/api"
+	"github.com/threefoldtech/rivine/pkg/cli"
+)
+
+// CommandLineClient extend for ERC20 commands
+type CommandLineClient struct {
+	*api.HTTPClient
+
+	RootCmd  *cobra.Command
+	ERC20Cmd *cobra.Command
+}
+
+// NewCommandLineClient creates a new CLI client, which can be run as it is,
+// or be extended/modified to fit your needs.
+func NewCommandLineClient(address, name, userAgent string) (*CommandLineClient, error) {
+	if address == "" {
+		address = "http://localhost:23111"
+	}
+	if name == "" {
+		name = "R?v?ne"
+	}
+	client := new(CommandLineClient)
+	client.HTTPClient = &api.HTTPClient{
+		RootURL:   address,
+		UserAgent: userAgent,
+	}
+
+	client.RootCmd = &cobra.Command{
+		Use:   os.Args[0],
+		Short: fmt.Sprintf("%s Client", strings.Title(name)),
+		Long:  fmt.Sprintf("%s Client", strings.Title(name)),
+		Run: Wrap(func() {
```

Comment:
I would not attach the version function to the RootCmd. It is ok that the root cmd has no function, that simply means if someone does try to run `bridgec` as-is that it will print the Usage text of the child commands, flags, global flags and description.